3.train_ebimage/1.random_forest/random-forest-pipeline.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Process the data using methods centered around random forest classifiers
+set -e
+
+# Run the notebook
+jupyter nbconvert --to=html \
+  --FilesWriter.build_directory=scripts/html \
+  --ExecutePreprocessor.kernel_name=python \
+  --ExecutePreprocessor.timeout=10000000 \
+  --execute random_forest.ipynb
```

Comment:
```suggestion
  --execute random_forest.ipynb
  
```

Adding new line to remove "no new line at bottom of file" indicator